src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -482,23 +482,6 @@ class G1RemSetScanState : public CHeapObj<mtGC> {
   }
 };
 
-G1RemSet::G1RemSet(G1CollectedHeap* g1h,
```

Comment:
Not completely sure why this moved but not `G1RemSet::initialize()` below.

Reply:
The destructor had to move to see the type and I obviously took the constructor but missed `initialize()`. Certainly makes sense to move that one too.